src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/RcaRuntimeMetrics.java

```diff
@@ -43,7 +43,12 @@
    * Metric tracking if RCA is enabled or disabled. We write a 0 if RCA is disabled and 1 if it is
    * enabled.
    */
-  RCA_ENABLED("RcaEnabled", "count", Collections.singletonList(Statistics.SAMPLE));
+  RCA_ENABLED("RcaEnabled", "count", Collections.singletonList(Statistics.SAMPLE)),
+
+  /**
+   * Metric tracking the actions published by the publisher that are persisted in sqlite.
+   */
+  ACTIONS_PUBLISHED("ActionsPublished", "count", Collections.singletonList(Statistics.NAMED_COUNTERS));
```

Comment:
Maybe we want to s/count/namedCount/ . Because this API supports it, we should get a counter for each proposed action ?

This is great that we are adding these metrics. It will give as a visibility into the tuning framework. Thanks for adding !

Reply:
Right. Changed this.